ethsigner/signer/hashicorp/src/main/java/tech/pegasys/ethsigner/signer/hashicorp/HashicorpSigner.java

```diff
@@ -24,17 +24,20 @@
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
+import java.util.stream.Stream;
 
 import io.vertx.core.Vertx;
 import io.vertx.core.http.HttpClient;
+import io.vertx.core.http.HttpClientOptions;
 import io.vertx.core.http.HttpClientRequest;
 import io.vertx.core.http.HttpMethod;
 import io.vertx.core.json.JsonObject;
+import io.vertx.core.net.PfxOptions;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.web3j.crypto.Credentials;
 
-public class HashicorpSignerFactory {
+public class HashicorpSigner {
```

Comment:
Why the rename? I think this is a factory after all, as a signer I would expect a signer to implement the TransactionSigner interface.

Reply:
yes...this was done by Sally earlier, I am renaming it back to factory.

Reply:
resolving (refactored to HashicorpVaultTransactionFactory).